package.json

```diff
@@ -5,7 +5,9 @@
   "description": "A browser plugin for tracking time for Gitlab merge-requests",
   "scripts": {
     "build": "node utils/build.js",
-    "start": "node utils/webserver.js"
+    "start": "node utils/webserver.js",
+    "format": "prettier --write 'src/css/options.css' 'src/css/popup.css' 'src/**/*.js' 'src/**/*.html'",
```

Comment:
Maybe better exclude bootstrap.css? I think it's more future-proof.
Maybe it's worth put bootstrap.css to a separate folder inside `css` dir (say, `lib`).

Reply:
Add lib folder